attendance/API_HR/c-hr.php

```diff
@@ -2434,6 +2432,42 @@ public function sendWelcomeMail($userID) {
 
     //end New Employee Welcome Email
 
+    //Leave of New Employee
+
+    public static function leaveOfNewEmployee($userID) {
+        $r_message3 = "";
+        $userInfo = self::getUserInfo($userID);
+        $date_of_joining = $userInfo['dateofjoining'];
+        $d = new DateTime($date_of_joining);
+        $d->modify('first day of this month');
+        $firstDay = $d->format('Y-m-d');
+        if($date_of_joining!=$firstDay) {            
+            $data = self::getDaysBetweenLeaves($firstDay, $date_of_joining);
+            $from_date = "";
+            foreach($data['data']['days'] as $var) {
+                if($var['type'] == 'working') {
+                    $from_date = $var['full_date'];
+                    break;
+                }
+            }
+            $to_date = $date_of_joining;
```

Comment:
you had passed to date  as first day of the employee on which employee is present